sdks/java/io/mongodb/src/test/java/org/apache/beam/sdk/io/mongodb/MongoDBGridFSIOTest.java

```diff
@@ -72,20 +70,21 @@
 import org.joda.time.Instant;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /** Test on the MongoDbGridFSIO. */
 public class MongoDBGridFSIOTest implements Serializable {
```

Comment:
Probably, my question is not related to this PR, but why `MongoDBGridFSIOTest` and `MongoDbIOTest` are `Serializable`? What is a reason for this?

Reply:
Legacy I will remove it.